support/cas-server-support-oidc/src/main/java/org/apereo/cas/web/controllers/OidcDynamicClientRegistrationEndpointController.java

```diff
@@ -123,8 +125,8 @@ protected OidcClientRegistrationResponse getClientRegistrationResponse(final Oid
         clientResponse.setClientName(registeredService.getName());
         clientResponse.setGrantTypes(Arrays.asList(OAuthGrantType.AUTHORIZATION_CODE.name().toLowerCase(),
                 OAuthGrantType.REFRESH_TOKEN.name().toLowerCase()));
-        clientResponse.setRedirectUris(Arrays.asList(registeredService.getServiceId()));
-        clientResponse.setResponseTypes(Arrays.asList(OAuthResponseType.CODE.name().toLowerCase()));
+        clientResponse.setRedirectUris(Collections.singletonList(registeredService.getServiceId()));
```

Comment:
Just wrote a tests for that. They are, but when deserialized using: ```List.class``` as the type, the list is of type ```ArrayList```.
We are fine, right?